common/rdr/Exception.cxx

```diff
@@ -54,33 +54,15 @@ SystemException::SystemException(const char* s, int err_)
 {
   strncat(str_, ": ", len-1-strlen(str_));
 #ifdef _WIN32
-  // Windows error messages are crap, so use unix ones for common errors.
-  const char* msg = 0;
-  switch (err) {
-  case WSAECONNREFUSED: msg = "Connection refused";       break;
-  case WSAETIMEDOUT:    msg = "Connection timed out";     break;
-  case WSAECONNRESET:   msg = "Connection reset by peer"; break;
-  case WSAECONNABORTED: msg = "Connection aborted";       break;
```

Comment:
Do we get decent messages for these now? Do you have some examples?

Reply:
If you try to a socket with no active server you get a translated message with the error code 10061 which is the equivalent of 'WSAECONNREFUSED'.